packages/squiggle-lang/__tests__/Distributions/Scoring/TwoScalars_test.res

```diff
@@ -0,0 +1,90 @@
+open Jest
```

Comment:
My guess is that this functionality really isn't necessary, so this file can/should be removed.

Reply:
removed.